tests/plugins/test_amp_plugins.py

```diff
@@ -201,10 +201,6 @@ def test_cpu_amp_precision_throws_error(tmpdir):
 
 
 @pytest.mark.skipif(not _TORCH_CPU_AMP_AVAILABLE, reason="Torch CPU AMP is not available.")
-@RunIf(
-    min_gpus=1,
```

Comment:
why did you remove `min_gpus=1,` here?

Reply:
@daniellepintz I suppose because 
given the setting `(precision="bf16", use_cpu=True)` and the check assert `isinstance(context_manager, torch.cpu.amp.autocast)` should pass regardless of whether we are on a cpu-only machien or one with gpus.

Reply:
makes sense, thanks!